rclpy/test/listener.py

```diff
@@ -43,7 +44,7 @@ def listener_cb(msg, message_name, received_messages):
     received_messages.append(msg)
 
 
-def listener(message_pkg, message_name, rmw_implemenatation, number_of_cycles=5):
+def listener(message_pkg, message_name, rmw_implementation, number_of_cycles):
```

Comment:
Yes it is, the test will be extended to use it in https://github.com/ros2/rclpy/pull/29